gnosis/safe-react
# Enhancement: cache owned Safes in localStorage

## What it solves
Owned safes on other networks are gone when you refresh the page.

## How this PR fixes it
Keeps them in localStorage.

I've added a wrapper around localStorage that does that. Plus a hook.

## How to test it
* Open the sidebar, switch between networks.
* Refresh the page
* Owned safes on all networks should be still in the sidebar
